libauv/Point2d center
float64 direction
float64 bearing
navig/MsgLocalPosition position